ledger/acctupdates_test.go

```diff
@@ -1746,7 +1744,7 @@ func TestAcctUpdatesSplittingConsensusVersionCommitsBoundary(t *testing.T) {
 
 	conf := config.GetDefaultLocal()
 	au, _ := newAcctUpdates(t, ml, conf)
-	defer au.close()
+	// au and ao are closed via ml.Close() -> ml.trackers.close()
```

Comment:
`ao` is no longer a variable.
```suggestion
	// accountUpdates and onlineAccounts are closed via: ml.Close() -> ml.trackers.close()
```